connector/github/github.go

```diff
@@ -411,12 +416,89 @@ func (c *githubConnector) user(ctx context.Context, client *http.Client) (user,
 		return u, fmt.Errorf("%s: %s", resp.Status, body)
 	}
 
-	if err := json.NewDecoder(resp.Body).Decode(&u); err != nil {
+	if err = json.NewDecoder(resp.Body).Decode(&u); err != nil {
```

Comment:
I changed this because my linter yells at me for shadowing `err` :sweat_smile: . `err` will be nil after this statement if not caught by the condition anyways, so I guess it's just a matter of style. Will change back.